src/wallet/wallet.cpp

```diff
@@ -1349,24 +1394,34 @@ void CWallet::SyncTransaction(const CTransaction& tx, const CBlock* pblock)
     MarkAffectedTransactionsDirty(tx);
 }
 
-void CWallet::MarkAffectedTransactionsDirty(const CTransaction& tx)
-{
+void CWallet::MarkAffectedTransactionsDirty(const CTransaction& tx) {
     // If a transaction changes 'conflicted' state, that changes the balance
     // available of the outputs it spends. So force those to be
     // recomputed, also:
-    BOOST_FOREACH(const CTxIn& txin, tx.vin)
+    BOOST_FOREACH(
+    const CTxIn &txin, tx.vin)
     {
         if (mapWallet.count(txin.prevout.hash))
             mapWallet[txin.prevout.hash].MarkDirty();
     }
-    for (const JSDescription& jsdesc : tx.vjoinsplit) {
-        for (const uint256& nullifier : jsdesc.nullifiers) {
+    for (const JSDescription &jsdesc : tx.vjoinsplit) {
+        for (const uint256 &nullifier : jsdesc.nullifiers) {
```

Comment:
Are these code-style changes automated by your IDE? They add a bit of noise to the PR.

Reply:
I'm experimenting with CLion at the moment.